src/parser/jobs/brd/changelog.js

```diff
@@ -1,10 +1,46 @@
 import React from 'react'
+import {List, Label} from 'semantic-ui-react'
 
 import {ActionLink} from 'components/ui/DbLink'
 import CONTRIBUTORS from 'data/CONTRIBUTORS'
 import ACTIONS from 'data/ACTIONS'
 
 export const changelog = [
+	{
+		date: new Date('2019-07-28'),
+		Changes: () => <>
+			Big <ActionLink showIcon={false} {...ACTIONS.PITCH_PERFECT}/> module update:
```

Comment:
Appreciate the detail, but it's causing the changelog to freak out pretty hardcore. Gonna guess it's a bad interplay between the css and the semantic UI elements. If you can get this to collapse properly, should be gucchi.

Reply:
It's because when the change log entry is collapsed it set's everything into the accordion title.  I am unsure how to fix that without changing the change log component itself